src/main/java/graphql/testrunner/service/CommandExecutorService.java

```diff
@@ -0,0 +1,89 @@
+package graphql.testrunner.service;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.List;
+import java.util.Objects;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import graphql.testrunner.util.TestRunnerException;
+
+import org.springframework.stereotype.Service;
+
+import static java.util.Arrays.asList;
+
+@Service
+public class CommandExecutorService {
+
+    public static final Logger LOGGER = Logger.getLogger(CommandExecutorService.class.getName());
+
+    public static final String GRAPHQL_DIR = "/app/graphql-java/";
+    public static final List<String> INSTALL_GIT = asList("sh", "-c", "apk add --no-cache git");
+    public static final List<String> BUILD_GRAPHQL_JAR = asList("sh", "-c", "RELEASE_VERSION=test-runner ./gradlew build");
+    public static final List<String> BUILD_GRAPHQL_JMH_JAR = asList("sh", "-c", "RELEASE_VERSION=test-runner-jmh ./gradlew jmhJar");
+
+
+    /**
+     * Executes the command in the default running directory of the application.
```

Comment:
A note detailing my personal preferences, you do not have to implement this:

When the code is clear, it doesn't require a comment. In fact, you should only write comments for unintuitive bits of code. In my opinion, the fewer comments the better.